tests/utils/plugin-stubs.nix

```
@@ -0,0 +1,364 @@
+/*
+  This file includes stub plugins with dependencies of various types.
+  It is primarily used in tests for the performance module.
+*/
+{ lib, pkgs }:
```

Better to use the `callPackages` pattern, IMO:

```suggestion
{
  lib,
  runCommand,
  neovim-unwrapped,
  neovimUtils,
  vimUtils, 
}:
```

Then import with `pkgs.callPackages ./utils/plugin-stubs.nix { }`